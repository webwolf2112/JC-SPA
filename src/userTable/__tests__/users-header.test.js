import React from 'react';
import { shallow } from 'enzyme';
import UsersHeader from '../users-header';

describe( 'Users Header ', () => {
    const defaultProps = {
        count: 5,
        handleCreateUser: jest.fn(),
    };

    beforeEach( () => {
        jest.clearAllMocks();
    });

    it('Should render correct', () => {
        const wrapper = shallow(
          <UsersHeader {...defaultProps} />,
        );
      
        expect(wrapper).toMatchSnapshot();
      });
});

