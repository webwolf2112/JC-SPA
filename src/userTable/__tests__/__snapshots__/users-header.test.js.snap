// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Users Header  Should render correct 1`] = `
<Fragment>
  <h1>
    Total Users: 
    5
  </h1>
  <button
    className="create"
    onClick={[MockFunction]}
  >
    Create New User
  </button>
</Fragment>
`;
